c++/Makefile.am

```diff
@@ -181,12 +184,14 @@ includecapnp_HEADERS =                                         \
   src/capnp/rpc.capnp.h                                        \
   src/capnp/rpc-twoparty.capnp.h                               \
   src/capnp/persistent.capnp.h                                 \
-  src/capnp/ez-rpc.h
+  src/capnp/ez-rpc.h                                           \
+  src/capnp/compat/json.h                                      \
```

Comment:
I am pretty sure this worked for me. It installed it in `${INSTAL}/include/compat/json.h`. I will check again. But I will rework this as suggested.